EOSatelliteImagery/schema.json

```diff
@@ -0,0 +1,102 @@
+               "enum":[
+                  "EOSatelliteImagery"
+               ]
+            },
+            "productBbox":{
+               "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons/properties/location"
+            },
+            "processedBbox":{
+               "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons/properties/location"
+            },
+            "analysisType":{
+               "type":"string",
+               "description":"Property. Entity's type of analysis applied"
+            },
+            "provider":{
+               "type":"string",
+               "description":"Property.  Model:'https://schema.org/Text'. The provider of the algorithm"
+            },
+            "contentInformation":{
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "properties":{
+                     "layer_name":{
+                        "type":"string"
+                     },
+                     "layer_categorization":{
+                        "type":"string",
+                        "enum":[
+                           "categorical",
+                           "numerical"
+                        ]
+                     },
+                     "values_explanation":{
+                        "type":"array",
+                        "items":{
+                           "type":"string"
+                        }
+                     }
+                  }
+               },
+               "description":"Property. An array that describes for each layer (e.g. band1) or entity name, the type of information (e.g. categorical, numerical) and an array with the explanation of the depicted values (e.g. [1:oil, 0:no oil])"
+            },
+            "localServerPath":{
+               "type":"string",
+               "description":"Property. Model:'https://schema.org/Text'. A mandatory text string used to declare the path that the output data layer is saved on the server"
+            },
+            "storageFormat":{
+               "type":"string",
+               "description":"Property. Entity's satorage format"
+            },
+            "operationalMode":{
+               "type":"string",
+               "description":"Property. Model:'https://schema.org/Text'. A text string used to declare the supported sensor modes if available"
+            },
+            "orbitNumber":{
+               "type":"number",
+               "description":"Property.  Model:'https://schema.org/Number'. The orbit number of tha satellite pass. All units are accepted in [CEFACT](https://www.unece.org/cefact.html) code"
+            },
+            "resultDescription":{
+               "type":"string",
+               "description":"Property. The description of the analysis outcome"
+            },
+            "sensingDate":{

```

Comment:
you have a timestamp in the examples which is not a iso 8601 like a format date-time. Consider changing the type to i.e. just string or the examples.